### ISSUE TITLE
Are all students necessarily going to be aware of what a "GUI" is?

### ISSUE THREAD

**Issue by dvanic** [datacarpentry/R-ecology-lesson#215]
The first thing that's presented in the Before we start material is that "R is not a GUI, and that’s a good thing". Can we assume that all learners will know what the abbreviation GUI is? 

Or would it perhaps be useful to paraphrase this as:

R is not a graphical user interface, and that’s a good thing!

or 

R does not have a graphical user interface, and that’s a good thing!

or even

Using R does not involve lots of pointing and clicking, and that’s a good thing!

**Comment by fmichonneau** [comment]
I like the 3rd suggestion with the explicit mention of pointing and clicking the best. The mention of GUI is actually confusing because the R executable downloaded from CRAN is called R GUI. 

Maybe rephrase:

"With R the results of your analysis does not rely on remembering a succession of pointing and clicking but instead in a series of written commands, and that's a good thing!"
